datadog_lambda/metric.py

```diff
@@ -32,6 +32,10 @@
     flush_in_thread = os.environ.get("DD_FLUSH_IN_THREAD", "").lower() == "true"
     lambda_stats = ThreadStatsWriter(flush_in_thread)
 
+enhanced_metrics_enabled = (
```

Comment:
why a tuple?

Reply:
It's not a tuple. Tuples have a comma in them. Without the comma it's just a way to put something on multiple lines.

Reply:
Oop, tysm :til: